Packs/Mimecast/Integrations/MimecastEventCollector/MimecastEventCollector.py

```diff
@@ -62,16 +62,16 @@ def prepare_headers(self, uri: str):  # pragma: no cover
             date=request_date,
             req_id=request_id,
             uri=uri,
-            app_key=self.options.app_key
+            app_key=self.options.app_key  # type: ignore[attr-defined]
```

Comment:
Why the changes here?
Please revert

Reply:
What about this? @maimorag

Reply:
it causes an error without it:
in the python file, there is an object(mimecastOptions) that is inherence from IntegrationOptions, but mypy doesn't recognize the fields that are in Mimecast options that were inherent with pydentic

Reply:
probably new validation were added since it was merged